src/heratape/tape_system.py

```diff
@@ -0,0 +1,86 @@
+import subprocess
+from .tapes import add_new_tapes_to_db,list_tapes_in_db
+
+def tape_in_drive(drivenum):                                                                                                                                                                                                                            
+    #input drivenum = 0 or 1                                                                                                                                                                                                                            
+    #returns: VolumeTag of tape in drive                                                                                                                                                                                                                
+    assert(drivenum in [0,1])                                                                                              
```

Comment:
maybe provide a helpful error message if this assert fails? Something like:
```suggestion
    assert drivenum in [0,1], "drivenum must be 0 or 1"                                                                                           
```